src/compose/images.ts

```diff
@@ -263,15 +282,10 @@ export async function getAvailable(): Promise<Image[]> {
 	);
 }
 
-// TODO: Why does this need a Bluebird.try?
-export function getDownloadingImageIds() {
-	return Bluebird.try(() =>
-		_(volatileState)
-			.pickBy({ status: 'Downloading' })
-			.keys()
-			.map(validation.checkInt)
-			.value(),
-	);
+export function getDownloadingImageIds(): number[] {
+	return _.keys(_.pickBy(volatileState)).map((i) =>
```

Comment:
Is this trying to match what was here before? If so it needs to be
```suggestion
	return _.keys(_.pickBy(volatileState, { status: 'Downloading' })).map((i) =>
```

Reply:
@CameronDiver to confirm, but I would say that without this extra then it is returning any image IDs irrespective of `status`.